src/app/pages/TopicPage/Curation/Subhead/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react';
+
+import Link from './index.styled';
+
+import { LeftChevron, RightChevron } from '../../icons';
+
+import { ServiceContext } from '../../../../contexts/ServiceContext';
+
+interface Props {
+  children: React.ReactNode;
```

Comment:
React has a built-in type called `PropsWithChildren` that you can wrap around your own type/interface like `PropsWithChildren<Props>`. This automatically includes the `children` type so you don't have to write it out.

Pretty sure under the hood it just declares `children: ReactNode` like you have here, but it keeps it global at least as `children` can be typed in many different ways, e.g. `JSX.Element, null, [], Array` etc.

Reply:
Nice, implemented that change now (hopefully correctly). Let me know if that satisfies!